solitude/settings/base.py

```diff
@@ -477,7 +477,8 @@
 }
 
 # The URL for a solitude proxy to zippy.
-ZIPPY_PROXY = os.getenv('SOLITUDE_ZIPPY_PROXY', '')
+ZIPPY_PROXY = os.getenv(
+    'SOLITUDE_ZIPPY_PROXY', 'http://solitude-auth:2603/v1/reference/')
```

Comment:
I think it's a little nicer to account for empty variable values which sometimes happens in docker setups. This would do it:
````
os.getenv('SOLITUDE_ZIPPY_PROXY') or 'http://solitude-auth:2603/v1/reference/'
````